toolset/setup/linux/languages/xsp.sh

```diff
@@ -7,18 +7,6 @@ RETCODE=$(fw_exists ${IROOT}/xsp.installed)
   source $IROOT/xsp.installed
   return 0; }
 
-# get
-git clone git://github.com/mono/xsp
-cd xsp
-git checkout e272a2c006211b6b03be2ef5bbb9e3f8fefd0768
-
-# build
-./autogen.sh --prefix=$MONO_HOME --disable-docs
-make
-sudo make install
-
-# cleanup
-cd ..
-rm -rf xsp
+sudo apt-get install -y mono-fastcgi-server
```

Comment:
I don't currently have access to linux/mac computer so it's unlikely I'll be able to change anything.

I suggest you just pick the xsp.sh change since xsp stands for multiple things and not all frameworks need it (thats why it's ok to be a separate dependency.. .although I have no problems if you install it with vanilla Mono)